### PR TITLE
DOC: Update release notes, add note about targeting v0.3.

### PR DESCRIPTION
**Pull request by danielballan**
The release notes will need more work, and obviously more content by v0.3, but this keeps them from falling too far behind. 

I pushed this and Casper's update to github pages. That 3D image looks great.

### CODE REVIEW

File: doc/releases/v0.3.0.txt [hunk 1]
```diff
@@ -11,12 +22,18 @@ Feature-Finding
...
+- Various performance improvements make feature-finding and trajectory-linking faster than ever, in spite of the increased functionality. (:issue:`203`, :issue:`204`)
+
+- More user-friendly error messages replace cryptic errors. (:issue:`166`).
+
 Bug Fixes
 ~~~~~~~~~
 
 - Fixed a bug in v0.2.3 and v0.2.4 that broke the ``circle_size`` parameter in ``annotate()``. (:issue:`169`, :issue:`170`)
+- Fixed a bug where subnetwork distances were not addded in quadrature in the python linker. (:issue:`212`). (The impact of this bug is not as large as one might expect. See issue link for details.)
+- Input images with a float datatype are internally converted to integer datatype for faster computaiton. They were converted to signed integers; now they are converted to unsigned. (:issue:`188`)
```
Comment (nkeim) [inline doc/releases/v0.3.0.txt]: Spelling of "computation"

Comment (nkeim) [inline doc/releases/v0.3.0.txt]: - Slow feature-finding with more recent versions of the numba package (0.16 and 0.17 specifically) is addressed.

File: doc/releases/v0.3.0.txt [hunk 2]
```diff
@@ -11,12 +22,18 @@ Feature-Finding
...
 
 API changes
 ~~~~~~~~~~~
 
-- The plot function ``annotate()`` now displays the image with the vertical axis inverted, to be consistent with the ``pims`` display function and ``plot_traj()``.
+- The plot function ``annotate()`` now displays the image with the vertical axis inverted, consistent with the ``pims`` display function and ``plot_traj()``.
```
Comment (nkeim) [inline doc/releases/v0.3.0.txt]: - Using trackpy to access pims functionality (e.g. ``trackpy.ImageSequence()``) is now deprecated; it still works but will generate a warning message. This capability will be removed from future versions of trackpy, in favor of accessing it in the pims package directly (e.g. ``pims.ImageSequence()``).

**Comment by nkeim** [comment]
Great idea!! See my line notes reflecting a couple other recent/imminent PRs.

**Comment by danielballan** [comment]
Rebased. I addressed @nkeim's comments awhile back -- should be good to merge now.

File: doc/releases/v0.3.0.txt [hunk 3]
```diff
@@ -2,8 +2,19 @@
...
+~~~~~~~~
+
+Release v0.3.0 adds many important enhancements and fixes. Feature location is extended to anisotropic 3D (and generally N-dimensional) features. The analysis and visualization tools now give first-class status to 3D work. A powerful new "adaptive search" capability gives fine-tuned control and detailed diagnostics of the trajectory-linking process. Several performance improvements make trackpy even faster.
+
+Enhancements
+~~~~~~~~~~~~
+
+- The feature-finding functionality was formerly limited to circular features, but now trackpy can locate anisotropic N-dimensional features. Their size is specified as a tuple of odd integers instead of a single odd integer. See the new 3D tutorial for details.
+
+.. Waiting for merge: - The plotting and analysis tools, many of which were limited to 2D, have been extended to 3D.
```
Comment (caspervdw) [inline doc/releases/v0.3.0.txt]: Already merged now

File: doc/releases/v0.3.0.txt [hunk 4]
```diff
@@ -2,8 +2,19 @@
...
+
+- A powerful new adaptive serach feature gives much richer control of the tracking (trajectory-linking) process, and it makes formerly intractable scenarios possible to solve in a reasonable time. See the new tutorials, "Advanced Linking: Subnetworks and search_range" and "Adaptive Search: Changing search_range on the Fly," for details.
```
Comment (caspervdw) [inline doc/releases/v0.3.0.txt]: search

**Comment by danielballan** [comment]
Bump. Addressed line comments, added Casper's recent work.

**Comment by nkeim** [comment]
Thanks! I guess linking diagnostics should get a line item under "Enhancements." Perhaps

- The linking functions can now collect data about how each particle was linked. See the tutorial "Obtaining Diagnostic Information from Linking."

The changes look great. I'm ready to merge now, or with this addition.

**Comment by danielballan** [comment]
@nkeim added

**Pull request merged**